lib/ansible/modules/cloud/amazon/aws_batch_job_definition.py

```diff
@@ -0,0 +1,526 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Jon Meran <jonathan.meran@sonos.com>
```

Comment:
Should follow copyright guideline http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright and have author info in module author field.